sections/header.liquid

```diff
@@ -350,41 +350,43 @@
     {%- endif -%}
 
```

Comment:
The search icon looks misaligned on mobile: https://screenshot.click/27-07-3oc3n-x1j46.png